java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToGCS.java

```diff
@@ -65,17 +80,40 @@ public void runTemplate() {
               + "in resources/conf/template.properties file or at runtime. Refer to jdbc/README.md for more instructions.");
     }
 
+    if (StringUtils.isNotBlank(concatedPartitionProps)
+        && ((StringUtils.isBlank(jdbcSQLPartitionColumn)
+                || StringUtils.isBlank(jdbcSQLLowerBound)
+                || StringUtils.isBlank(jdbcSQLUpperBound))
+            || StringUtils.isBlank(jdbcSQLNumPartitions))) {
+      throw new IllegalArgumentException(
+          "Required parameters for JDBCToGCS not passed. "
+              + "Set all the sql partitioning parameters together"
+              + "in resources/conf/template.properties file or at runtime. Refer to jdbc/README.md for more instructions.");
+    }
+
     LOGGER.info(
         "Starting JDBC to GCS spark job with following parameters:"
             + "1. {}:{}"
             + "2. {}:{}"
-            + "3. {}:{}",
+            + "3. {}:{}"
+            + "4. {}:{}"
+            + "5. {}:{}"
+            + "6. {}:{}"
+            + "7. {}:{}",
         JDBC_TO_GCS_OUTPUT_LOCATION,
-        GCSOutputLocation,
+        gcsOutputLocation,
         JDBC_TO_GCS_WRITE_MODE,
-        GCSWriteMode,
+        gcsWriteMode,
         JDBC_TO_GCS_JDBC_URL,
```

Comment:
Printing JDBC URL in logs printing password also in logs. May be we can remove password and print or simply ignore printing jdbc url for now. It's okay to handle this in a new issue too.

Reply:
Good point thanks. Removed it.